pages/linux/runuser.md

```diff
@@ -0,0 +1,23 @@
+# runuser
+
+> Run commands as a specific user and group IDs. Does not ask for password. Only useful when run as the root user.
+
+- Run command as a different user:
+
+`runuser {{user}} -c '{{command}}'`
+
+- Run command as a different user and group:
+
+`runuser {{user}} -g {{group}} -c '{{command}}'`
+
+- Start a login shell as a specific user:
+
+`runuser {{user}} -l`
+
+- Specific a shell for running or login instead of default shell:
```

Comment:
```suggestion
- Specify a [s]hell for running instead of default shell (also works for login):
```
Typo. Also, since the command we show only does one of the two things, let's reword to say it *can* also be used for login.

Reply:
Agree, thanks for reviewing.